src/styles/theme/styled.d.ts

```diff
@@ -1,5 +1,5 @@
 import {} from 'styled-components';
-import theme from './theme';
+import theme from '.';
```

Comment:
If you're looking for that kind of refactoring, would be great to update other the same imports